lib/collnew/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "collnew"
+version = "1.0.0"
+authors = ["Vector Contributors <vector@timber.io>"]
+edition = "2018"
+description = "Macros for creating new Collection"
+publish = false
+license = "MIT"
+
+[features]
+default = ["btreemap"]
+btreemap = []
```

Comment:
Specifying the collection type via features doesn't seem very flexible. Was an alternative design considered?

Reply:
No, I did not thought about other ways. I think I'll remove `features` for now because `collnew` have only `btreemap` macro.